lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -119,31 +124,37 @@ class FeatureCard extends React.Component {
         }, 1000);
 
         this.setState({ //TODO: Fancy animation
-          signup_animation: 'tada'
+          signupAnimation: 'tada'
         });
       }
     }));
   }
 
   viewAnother() {
-    let newIndex = this.state.current_campaign_index + 1;
-    if (newIndex >= this.state.campaigns.length) {
-      newIndex = 0;
-    }
+    // Remove current campaign
+    this.state.campaigns.splice(0, 1);
```

Comment:
This seems pretty confusing (and potentially harder to change, for example if we we wanted a "go back" button in the future). What's the advantage over using an index to pick which campaign to display?